app/serializers/build_v2_fallback.js

```diff
@@ -37,11 +37,7 @@ export default V2FallbackSerializer.extend({
   normalize: function (modelClass, resourceHash) {
     // TODO: remove this after switching to V3 entirely
     let type = resourceHash['@type'];
-    let permissionsHash = resourceHash['@permissions'];
     let commit = resourceHash.commit;
-    if (permissionsHash) {
-      resourceHash.permissions = permissionsHash;
-    }
```

Comment:
Could you please revert this change? My previous comments for this change was to pull it from the upstream (which you've already done) instead of re-introducing it in the branch. With these lines removed the `permissions` property will never work on `build` model.

Reply:
Done.

Reply:
Thank you! 👍